api/src/opentrons/protocol_engine/state/labware.py

```diff
@@ -281,6 +301,61 @@ def get_definition_by_uri(self, uri: LabwareUri) -> LabwareDefinition:
                 f"Labware definition for matching {uri} not found."
             ) from e
 
+    def find_labware_params(
```

Comment:
niptick? not sure about this name
```suggestion
    def get_labware_definition_params(
```

Reply:
"Find" feels appropriate to me here because it's an $O(n)$ search. To me, "get" connotes a cheap $O(1)$ retrieval.

And instead of "definition," what about `find_labware_load_params()` (or `find_load_labware_params()`) to match the name of the return type?